exercises/crypto-square/Cargo.toml

```diff
@@ -0,0 +1,6 @@
+[package]
+name = "crypto-square"
+version = "0.1.0"
```

Comment:
given that https://github.com/exercism/problem-specifications/blob/master/exercises/crypto-square/canonical-data.json exists, normally I would say this should be 2.0.0 and using the test cases from this file. However, since https://github.com/exercism/problem-specifications/issues/750 exists, I wish to exempt crypto-square from following problem-specifications in its current form.